src/Paket.Core/TemplateFile.fs

```diff
@@ -6,6 +6,80 @@ open System.IO
 open System.Text.RegularExpressions
 open Paket.Rop
 open Paket.Domain
+
+module private TemplateParser =
+    type private ParserState =
+        {
+            Remaining : string list
+            Map : Map<string, string>
+            Line : int
+        }
+
+    let private single = Regex("^(\S+)\s*$", RegexOptions.Compiled)
```

Comment:
It's not a bug - it works correctly. It is pretty bad practice though :) . The ``@"..."`` version would be a much better idea, my bad.